external/mve11/mve_run.m

```diff
@@ -1,286 +0,0 @@
-function [x,E] = mve_run(A,b,x0)
```

Comment:
@Bounciness , yes, that would be much better. In `external/`, we have code that we do not maintain, but provide for convenience.

May you make the changes? I gave you write access on my fork. Simply push to the branch `pr-310` and it will show up here in the PR. The commands are:
```
git clone https://github.com/laurentheirendt/cobratoolbox.git
git checkout pr-310
# do your changes
git add [yourFiles]
git commit -m "[yourMessage]"
git push origin pr-310
```